.chloggen/splunk-otel-python.yaml

```
@@ -0,0 +1,13 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: operator
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use quay.io/signalfx/splunk-otel-instrumentation-python image for auto-instrumentation of Python applications with the operator.
+# One or more tracking issues related to the change
+issues: [1965]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  - To customize the image for Python auto-instrumentation, set the configuration option `instrumentation.python.image`.
```

```suggestion
  - To use a different image for Python auto-instrumentation, set the configuration option `instrumentation.python.image`.
```
Nit: "To customize the image" sounds like you could actually modify the image itself, not use a different image.